spec/cocaine/command_line/output_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Cocaine::CommandLine::Output do
+  it 'holds an input and error stream' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.